### DISCUSSION TITLE
关于此项目的功能请求，涉及并列查看和筛选排序等。

### DISCUSSION THREAD

**Discussion by huajingyu** [CollegesChat/university-information#277] [Enhancement]
建议此项目增加排序和筛选功能，方便直接筛选出符合条件的学校。

以下是为方便筛选、统计、排序而提出的数据库优化建议：
对于此项目的一些封闭式问题（比如“是否允许带电脑”），可以以布尔值的方式保存，而非文本。（因为文本可能会出现尴尬的情况，比如结尾加句号和未加句号的否会被视为不同的值。）只有备注才需要添加文本。
对于部分开放式问题，可以优化一下保存的方式。比如宿舍热水供应时间，可以将开始时间和结束时间以数字的方式保存。比如位置，可以以经纬度的方式保存。

以下是其它想法：
将来是否有可能创建幼儿园/小学/初中/高中/中职生活质量指北？
将来是否会纳入中国大陆境外的学校？

**Comment by jahn2007** [comment]
用bool保存不好吧？我随便说的，比如有些学校是申请后可带？这怎么保存？
用字符串保存可以增加任何回答者想增添的细节

**Reply by huajingyu** [reply]
> 用bool保存不好吧？我随便说的，比如有些学校是申请后可带？这怎么保存？ 用字符串保存可以增加任何回答者想增添的细节

那也许应该设置成多个选项，先是已知选项，比如完全允许或完全禁止。然后是介于这两者中间，当选择两者中间时，增加一个文本字段。
总之希望尽可能容易检索。
